### ISSUE TITLE
Change Accessibility Icon and fix flickering of the accessibility bar 

### ISSUE THREAD

**Issue by gggodhwani** [NationalDataPlatform/NationalDataPlatform#112]
**Describe the bug**

- The accessibility icon is not appropriate at the moment, here is the link which looks more appropriate: https://thenounproject.com/icon/accessibility-1872/ 
- The white coloured accessibility bar is flickering at the moment when you hover over the icon
- Colour change option is missing from the accessibility bar
- Sitemap link is missing from it
- Please generate the Sitemap.xml for the platform for all URLs

**Comment by PixeledCode** [comment]
closed with #115
